docs/api-testing/composer/io-components.md

```diff
@@ -17,9 +17,10 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-I/O request components, available in your tests' **Compose** tab (aka Composer), enable you to perform the I/O operations `GET`, `POST`, `PUT`, `PATCH`, and `DELETE`. This guide describes each component and shows you how to add them to tests.
+I/O request components enable you to perform the I/O operations `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, `Github`, `Contract Test`, and `File DataSource`. This guide describes each component and shows you how to add them to tests. To learn how to access the components and create a test using the Composer see [Writing API Tests with the Composer](/api-testing/composer/).
```

Comment:
@LoryApiFortress, Updated as you noted -  I think it is good to link to the Composer topic for steps to access the components.